assets/js/modules/thank-with-google/components/setup/SetupForm.test.js

```diff
@@ -39,8 +39,9 @@ describe( 'SetupForm', () => {
 
```

Comment:
`defaultSettings` above needs updating as well, it currently contains `products` and `revenueModel`.